lib/statsd/instrument.rb

```diff
@@ -94,8 +94,8 @@ def self.duration
     def statsd_measure(method, name, *metric_options)
       add_to_method(method, name, :measure) do
         define_method(method) do |*args, &block|
-          metric_name = StatsD::Instrument.generate_metric_name(name, self, *args)
-          StatsD.measure(metric_name, *metric_options) { super(*args, &block) }
+          key = StatsD::Instrument.generate_metric_name(name, self, *args)
+          StatsD.measure(key, *metric_options) { super(*args, &block) }
```

Comment:
Why are we fine the the args splat here, but avoid doing that in `statsd_distribution`?

Reply:
Ugh, I missed this.

Reply:
This is why we need #181 :P